soccer/gameplay/plays/offense/adaptive_formation.py

```diff
@@ -145,6 +145,8 @@ def __init__(self):
 
     @classmethod
     def score(cls):
```

Comment:
I think it's lowest. We need to verify and rename the 'score' variable though. But the score does in some way influence which play is selected :)